src/devices/emax.c

```diff
@@ -0,0 +1,233 @@
+/** @file
+    First version was for Altronics X7064 temperature and humidity sensor.
+    Then updated by Profboc75 with Optex 990040 (Emax full Weather station rain gauge/wind speed/wind direction ... ref EM3390W6 )
+    
+    Copyright (C) 2022 Christian W. Zuckschwerdt <zany@triq.net>
+    based on protocol decoding by Thomas White
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+#include "decoder.h"
+
+/**
+Altronics X7064 / Emax temperature and humidity sensor.
```

Comment:
Should be "Emax EM3390W6 / Altronics X7064 weather sensor."

Reply:
Same as previous message, 
"Emax W6 and rebrand .... "